docs/install-mlrun-ce/storing-artifact-in-aws-s3-storage.md

```diff
@@ -0,0 +1,62 @@
+# Storing artifacts in AWS S3 storage
+
+MLRun CE uses a MinIO service as a shared storage for artifacts, and accesses it using S3 protocol. This means that
+any path that begins with `s3://` is automatically directed by MLRun to the MinIO service. The default artifact
+path is also configured as `s3://mlrun/projects/{{run.project}}/artifacts` which is a path on the `mlrun` bucket in the
+MinIO service.
+
+To store artifacts in AWS S3 buckets instead of the local MinIO service, these configurations need to be overridden to 
+make `s3://` paths lead to AWS buckets instead.
+
+```{admonition} Note
+These configurations are only required for AWS S3 storage, due to the usage of the same S3 protocol in MinIO. For other
+storage options (such as GCS, Azure blobs etc.) only the artifact path needs to be modified, and credentials need to
+be provided.
+```
+
+## Setting up S3 credentials and endpoint
+
+Set up the following project-secrets (refer to [**Data stores**](../store/datastore.md) and [**Project secrets**](../secrets.md#mlrun-managed-secrets)) 
+for any project used:
+
+* `AWS_ACCESS_KEY_ID`, `AWS_SECRET_ACCESS_KEY` &mdash; S3 credentials
+* `AWS_ENDPOINT_URL_S3` &mdash; the AWS S3 endpoint to use, depending on the region. For example: 
+    ``` console
+    AWS_ENDPOINT_URL_S3 = https://s3.us-east-2.amazonaws.com/
+    ```
+    **Note**: `S3_ENDPOINT_URL` is deprecated as of v1.10.0 and will be removed in v1.12.0. Use `AWS_ENDPOINT_URL_S3` instead.
+
+## Disabling auto-mount
+
+Before running any MLRun job that writes to an S3 bucket, make sure auto-mount is disabled for it, since by default
+auto-mount adds S3 configurations that point at the MinIO service (refer to 
+[**Function storage**](../runtimes/function-storage.md) for more details on auto-mount). This can be done in one
+of following ways:
+
+* Set the client-side MLRun configuration to disable auto-mount. This disables auto-mount for any function run
```

Comment:
* Set the client-side MLRun configuration to disable auto-mount. This disables auto-mount for any functions you subsequently run: